generators/python/src/fern_python/generators/sdk/client_generator/endpoint_function_generator.py

```diff
@@ -709,9 +709,9 @@ def _get_headers_for_endpoint(
             if literal_header_value is not None and type(literal_header_value) is str:
                 headers.append((header.name.wire_value, AST.Expression(f'"{literal_header_value}"')))
             elif literal_header_value is not None and type(literal_header_value) is bool:
-                headers.append((header.name.wire_value, AST.Expression(f"{literal_header_value}")))
+                headers.append((header.name.wire_value, AST.Expression(f'"{str(literal_header_value).lower()}"')))
```

Comment:
boolean headers are pretty non-standard, but we need to stringify somehow

Reply:
str() makes sense to me